docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -148,6 +148,8 @@ Preparing For a Release
 
    Go to the `BuildBot web status`_ and force a build on the just-created branch.
 
+   In addition, review the link-check step of the documentation builder to ensure that all the errors are expected.
```

Comment:
This makes me sad, but a red builder makes me sadder.

I think the trade-off of doing this during the release is worth it for the moment. Long-term, we should fix the link-check. (I guess a ticket should be filed).